pom.xml

```diff
@@ -247,6 +249,21 @@
          </properties>
       </profile>
       <profile>
+         <id>redis</id>
+         <activation>
+            <property>
+               <name>!no-redis</name>
```

Comment:
This parameter should get added to "Building binaries" documentation, both as parameter and as part of directory structure